server/webSocketServer/UserStatus/statusTimer.js

```diff
@@ -0,0 +1,42 @@
+const statuses = require('./statuses');
+const { createOtherUserEditedInfoMessage } = require('../messageCreator');
+
+//This actually should be called statusTimerInitiatorCreator but it's a bit much
+const statusTimer = (userContainer, timeToChangeStatus) => {
```

Comment:
Nice job
Just for you to know, it could be done much easier with Idle Timer in the client side.

Reply:
Hmmm, didn't think about making a client side thing... nice job lol